homeassistant/components/device_tracker/owntracks.py

```diff
@@ -49,6 +52,8 @@ def owntracks_location_update(topic, payload, qos):
 
         # Block updates if we're in a region
         with LOCK:
+            _LOGGER.error("REGIONS_ENTERED %s",REGIONS_ENTERED)
+            _LOGGER.error("MOBILE_BEACONS_ACTIVE %s",MOBILE_BEACONS_ACTIVE)
```

Comment:
Still walking up and down the street to confirm all the right things are happening.

Will remove when done!